.github/workflows/release.yml

```
@@ -0,0 +1,56 @@
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --target ${{ matrix.target }}
+
+      - name: Package binary (Unix)
+        if: matrix.os != 'windows-latest'
+        run: |
+          tar -czvf ${{ matrix.artifact_name }}.tar.gz -C target/${{ matrix.target }}/release askman
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.11)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)